CODE_OF_CONDUCT.md

```diff
@@ -105,13 +102,11 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-the email addresses listed above in the
-[Reporting](#report-an-issue-in-a-project) and
-[Escalation](#escalate-an-issue) sections.
+the email addresses listed above in the [Reporting](#report-an-incident) section.
 All complaints will be reviewed and investigated promptly and fairly.
 
 All community leaders are obligated to respect the privacy and security of the
-reporter of any incident.
+harmed individuals and the reporter(s) of any incident.
```

Comment:
after this is landed i'll remove all the hard wraps, consistent with #1116.

Reply:
Thank you!